build.sbt

```diff
@@ -188,7 +185,7 @@ def youngestForwardCompatible(subProj: String) =
 // Uncomment after release.
 //  Some(subProj)
 //    .filterNot(unreleasedModules.contains(_))
```

Comment:
no, we need to uncomment after each version bump. Deleting seems like a good way to keep making mima mistakes like the scalding one.